src/privatesend-util.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2014-2017 The Dash Core developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef PRIVATESENDUTIL_H
+#define PRIVATESENDUTIL_H
+
+#include "wallet/wallet.h"
+
+class CKeyHolder
+{
+private:
+    CReserveKey reserveKey;
+    CPubKey pubKey;
+public:
+    CKeyHolder(CWallet* pwalletIn);
```

Comment:
Disable copy/assign and allow move by using:
CKeyHolder(CKeyHolder&&) = default;
CKeyHolder& operator=(CKeyHolder&&) = default;